tsconfig.json

```diff
@@ -11,6 +11,8 @@
     { "path": "packages/remix-dev" },
     { "path": "packages/remix-express" },
     { "path": "packages/remix-netlify" },
+    // Commenting out for now because of https://github.com/remix-run/remix/pull/3104#issuecomment-1331779471
+    // { "path": "packages/remix-netlify-edge" },
```

Comment:
Can we somehow figure out a way to still typecheck `@remix-run/netlify-edge` or is that currently impossible because of https://github.com/microsoft/TypeScript/issues/37582?

Reply:
My plan was that once #4715 lands we can add the netlify-edge path to the glob and it would typecheck it too

Reply:
I'd delete this line rather than commenting it out. Once .ts extension land we can add it back so we can generate .d.ts files, but for now we can use Deno typechecking instead.

Reply:
@ascorbic #4715 is merged now, so can you please add the typecheck for `@remix-run/netlify-edge`?

Reply:
I'll go ahead and do that @MichaelDeBoey as @ascorbic is done for the day.

Reply:
Done via https://github.com/nickytonline/remix/pull/2